### DISCUSSION TITLE
Android Port

### DISCUSSION THREAD

**Discussion by SilentMystification** [Drewol/unnamed-sdvx-clone#523] [Ideas]
Since there is already a Linux and an embedded port of this game for RPis, how feasible would it be to port this to Android? With a mini controller and a USB C -> USB C (or whatever port the mini controller has) it would be trivial to plug in a mini con and have a portable on the go USC setup

**Comment by Drewol** [comment]
@defvs recently started a porting attempt over at https://github.com/defvs/usc-android but I cannot really contribute to that myself other than answering quick questions about the usc codebase.

**Reply by defvs** [reply]
I moved it to my fork of this repo. I needed the sources and a bunch of other stuff so it was easier to just fork.
See https://github.com/defvs/unnamed-sdvx-clone/tree/feat/android-port

Busy with life currently but I'll be working on it soon.
